test_data/IS0401/dns_records.zone

```diff
@@ -3,84 +3,84 @@ mocks.{{ domain }}.	IN	A	{{ ip_address }}
 timeout.{{ domain }}.	IN	A	192.0.2.1
 
 ; There should be one PTR record for each instance of the service you wish to advertise.
-_nmos-registration._tcp	PTR	reg-api-5101-ver._nmos-registration._tcp
-_nmos-register._tcp	PTR	reg-api-5101-ver._nmos-register._tcp
-_nmos-query._tcp	PTR	qry-api-5101-ver._nmos-query._tcp
-_nmos-registration._tcp	PTR	reg-api-5101-proto._nmos-registration._tcp
-_nmos-register._tcp	PTR	reg-api-5101-proto._nmos-register._tcp
-_nmos-query._tcp	PTR	qry-api-5101-proto._nmos-query._tcp
+_nmos-registration._tcp	PTR	reg-api-101-ver._nmos-registration._tcp
```

Comment:
Why won't the service instance names like `reg-api-101...` conflict between multiple test suite instances running on the same host?

Reply:
The DNS server can only run on port 53 as I don't believe Nodes could be configured to find it anywhere else. As such you're only ever going to be able to run one unicast DNS instance on a host at a time.